### PR TITLE
Tweaks the Lugol

### PR DESCRIPTION
**Pull request by goober3**
## About The Pull Request

This PR:
Adds a mining job to the Lugol
Adds 5 more uranium, 30 more plastic, 50 glass sheets, and replaces the titanium and plasma sheets with plastitanium

The RCD crate has large ammo now
And the Project Leader now has a pair of binoculars to watch their ship explode to bits from afar 

## Why It's Good For The Game

Long and resource-intensive engineering projects are now easier, wall shutters are bad.

## Changelog

:cl:
tweak: The Lugol's gear and jobs have been increased to facilitate engineering projects better. The engine room has been redesigned.
fix: There is no longer a wall shutter in the Lugol's engine room.
/:cl:

### CODE REVIEW

**Comment by goober3** [comment]
for my next update, i'll replace all of the walls with explosive plastitanium!

**Review by Erikafox — APPROVED** [review]

**Review by spockye — APPROVED** [review]
looks good!

**Comment by goober3** [comment]
pranking the maptainers by updating this post-review

**Comment by meemofcourse** [comment]
illegal

File: _maps/configs/syndicate_lugol.json [hunk 1]
```diff
@@ -24,6 +24,10 @@
 			"outfit": "/datum/outfit/job/bartender/syndicate",
 			"slots": 1
 		},
+		"Resource Haulier": {
```
Comment (Erikafox) [inline _maps/configs/syndicate_lugol.json]: I'm gonna donk you for this and the Potation whatever. 

Weird Job Names are bad for people who don't already know what the job is, and this is much better than potation specialist but you should change both of them anyays

**Pull request merged**